src/dataflow/bin_script.rs

```diff
@@ -43,7 +46,20 @@ fn save<P: AsRef<Path>>(data: String, directory: P, command_name: String) -> Res
             .map_err(|e| Error::SaveError(command_name.clone(), e.to_string()))?;
     }
     let script_path = dir.join(command_name.clone());
-    fs::write(script_path, data)
+    let mut script_file = {
+        let mut oo = fs::OpenOptions::new();
+        oo.create(true).truncate(true).write(true);
+        #[cfg(unix)]
+        let oo = {
+            use std::os::unix::fs::OpenOptionsExt;
+            oo.mode(0o731)
```

Comment:
Can we just add executable perms? (without overwriting the other perms) Thoughts?

Reply:
That's a good idea, I'll look in to it